mobile/src/navigation/navigation.epics.ts

```diff
@@ -29,6 +35,12 @@ const openDepartment$ = navigateEpic<OpenDepartmentAction>(
     'CurrentDepartment'
 );
 
+//----------------------------------------------------------------------------
+const loadDepartment$ = (action$: ActionsObservable<OpenDepartmentAction>) => action$.pipe(
+    ofType(NavigationActionType.openDepartment),
+    map(x => loadEmployeesForDepartment(x.params.departmentId)),
+);
```

Comment:
Not sure if navigation.epics is a good place for this one

Reply:
Me too. But what is a good place then?